src/service/impl/journey.ts

```diff
@@ -36,9 +26,7 @@ export default (
     },
     async getTripPatterns(query) {
       try {
-        batchedPublisher.publish(Buffer.from(JSON.stringify(query)), {
-          environment: ENV
-        });
+        await publisher.getTripPatterns(query);
```

Comment:
Usikker på om det burde være `await` her. Vil vi faktisk vente med å hente tripPattern for statistikk? Eller burde det være fire-and-forget? Hva skjer om det feiler her?

Reply:
Topic er batched, så det vil i de fleste tilfeller være øyeblikkelig, men jeg er enig i at det føles galt å bruke `await` her.

Kanskje `analytics` kunne koblet på en `catch` og skrevet til en logg om det feiler.